web/frontend/src/lib/components/robot-client.svelte

```diff
@@ -491,27 +500,56 @@ if (supportedAuthTypes.length === 0) {
 {#if $connectionStatus === 'connected' || $connectionStatus === 'reconnecting'}
   <slot />
 {:else}
-  {#each supportedAuthTypes as authType (authType)}
-    <div class="px-4 py-3">
-      <span>{authType}: </span>
-      <div class="w-96">
-        <input
-          bind:value={password}
-          disabled={$connectionStatus === 'connecting'}
-          class="
+  <div class="flex bg-[#f7f7f8] min-h-[100vh]">
```

Comment:
would be nice to add the viam logo here but didn't seem like the svg already existed in rdk? how would i add something like that

Reply:
oh great! thanks! looks like it works: 
<img width="521" alt="Screenshot 2023-12-04 at 10 03 56 AM" src="https://github.com/viamrobotics/rdk/assets/19638145/df69edae-207b-4d11-a3dd-0dca9505e5b5">